lib/src/Base/Stat/SampleImplementation.cxx

```diff
@@ -2152,26 +2153,59 @@ void SampleImplementation::exportToCSVFile(const FileName & filename,
       if (isBlank) csvFile << separator << "\"NoDescription\"";
       else csvFile << separator << "\"" << description[i] << "\"";
     }
-    csvFile << std::endl;
+    csvFile << "\n";
```

Comment:
Nope, this is a common misunderstanding, std::endl does not deal with CR+LF; \n is automatically converted to \r\n on Windows when files is open as a text file.